tests/cli/output/help.txt

```diff
@@ -69,6 +69,6 @@ Note: Use comments containing ' *INDENT-OFF*' and ' *INDENT-ON*' to disable
       processing of parts of the source file (these can be overridden with
       enable_processing_cmt and disable_processing_cmt).
 
-There are currently 650 options and minimal documentation.
+There are currently 651 options and minimal documentation.
```

Comment:
I have no idea how come two comments is adding a new option.

Reply:
I assume this is borked in master, and no one has made a PR specifically to fix it...

Reply:
It is now OK.